jdbc-handler/pom.xml

```diff
@@ -29,6 +29,7 @@
 
   <properties>
     <hive.path.to.root>..</hive.path.to.root>
+    <commons-dbcp.version>1.4</commons-dbcp.version>
```

Comment:
what is this?

Reply:
just google it: https://commons.apache.org/proper/commons-dbcp/

Reply:
Yeah it's strange that it's missing, but when I try to build it on my laptop, it was complaining that this dependency was missing. They even have the version listed in the top level `pom.xml` - https://github.com/6si/hive/blob/6si-main/pom.xml#L132, but this is the module that actually needs it and it's not listed there.